examples/theme.toml

```diff
@@ -42,6 +42,31 @@ bright_magenta = "#ff79c6"
 bright_cyan = "#8be9fd"
 bright_white = "#ffffff"
 
+[[themes]]
+name = "my_dracula"
```

Comment:
nit: can you name it `dracula2` or something? Can also add a small comment explaining the difference between this and the previous `dracula`